bre/bre.py

```diff
@@ -1,2 +1,79 @@
+import csv
+
+
+class Story:
+    def __init__(self, pages, current_page):
+        self.pages = pages
+        self.current_page = current_page
+
+
+class Page:
+    def __init__(self, page, text, choices):
+        self.page = page
+        self.text = text
+        self.choices = choices
+
+
+class Choice:
+    def __init__(self, choice, next_page):
+        self.choice = choice
+        self.next_page = next_page
```

Comment:
As above, I'd like to change this to next_page_id to make it clear that it's the page number rather than the page object that goes here. 

But we can change it in a future PR

Reply:
Sounds good for the next PR!